src/flash_internal.h

```diff
@@ -92,6 +94,23 @@ struct Flash_fwd_params : public Qkv_params {
 
     bool is_causal;
     int sm = 80;
+
+    /****** new class members in hopper ******/
+    int total_q, total_k;
+    uint32_t scale_softmax_log2_half2;
+    int* __restrict__ blockmask;
+    // The dropout probability (probability of keeping an activation).
+    float p_dropout;
+    uint8_t p_dropout_in_uint8_t;
+
+    // Scale factor of 1 / (1 - p_dropout).
+    float rp_dropout;
+    float scale_softmax_rp_dropout;
```

Comment:
We don't use dropout, so remove these params.

You also need to modify the kernel to remove dropout.